src/components/ui/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import * as AccordionPrimitive from "@radix-ui/react-accordion";
```

Comment:
Just as a question: What does the Nitro Accordion provide that the @radix-ui/react-accordion does not? Is this all about making it match the NitroUI style or is this just the best practice for bringing in the @radix-ui components?

Reply:
So I think of the radix accordion as owning the behavior, and nitro ui as owning the styling. In this case we're not adding in much in the way of style, but it let's us quickly modify that in the future.

an example of how this works for the other ui components is through `text-brand` and `text-noir` utility styles that nitro exposes. This lets consumers customize their [tailwind config](https://github.com/nitro-bio/nitro-ui/blob/well-annotation/tailwind.config.js#L25-L26) and change the look and feel of the components.